mlflow/server/js/src/experiment-tracking/pages/prompts/components/PromptContentPreview.tsx

```diff
@@ -124,13 +141,22 @@ export const PromptContentPreview = ({
           overflow: 'auto',
         }}
       >
-        <Typography.Text
-          css={{
-            whiteSpace: 'pre-wrap',
-          }}
-        >
-          {value || 'Empty'}
-        </Typography.Text>
+        {promptType === PROMPT_TYPE_CHAT && parsedMessages ? (
+          parsedMessages.map((msg: any, index: number) => (
+            <div key={index} css={{ marginBottom: theme.spacing.sm }}>
+              <Typography.Text>{msg.role}:</Typography.Text>{' '}
+              <Typography.Text css={{ whiteSpace: 'pre-wrap' }}>{msg.content}</Typography.Text>
+            </div>
+          ))
```

Comment:
you can try experimenting with using `ModelTraceExplorerChatMessage` to render this, since the prompt chat type should be compatible with the trace UI chat type:

```suggestion
          parsedMessages.map((msg: PromptChatMessage, index: number) => (
            <ModelTraceExplorerChatMessage key={index} message={msg} />
          ))
```